src/statsmodels/formula.jl

```diff
@@ -57,7 +57,9 @@ end
 ## an expression or a formula
 function allvars(ex::Expr)
     if ex.head != :call error("Non-call expression encountered") end
-    [[allvars(a) for a in ex.args[2:end]]...]
+    cc=Symbol[]
+    for i in ex.args[2:end] cc=vcat(cc,allvars(i)) end
```

Comment:
You should use `push!` here instead of `vcat`. In general you should never write a loop that contains a call to `vcat`.